certbot/certbot/crypto_util.py

```diff
@@ -204,7 +203,8 @@ def import_csr_file(
         except ValueError:
             raise errors.Error("Failed to parse CSR file: {0}".format(csrfile))
 
-    domains = _get_names_from_subject_and_extensions(csr.subject, csr.extensions)
+    # pylint: disable=protected-access
+    domains = acme_crypto_util._get_names_from_subject_and_extensions(csr.subject, csr.extensions)
```

Comment:
Actually, your other comment made me realize -- we probably shouldn't be using a protected method in certbot, because of that exact reason of using newer acme with older certbot. I agree it makes sense to move it to acme, but that probably does mean it should be made public.

Reply:
Done